### PR TITLE
Issue 8 variable combinations

### PR DESCRIPTION
**Pull request by davidplumb**

### CODE REVIEW

File: tables/variable_combinations.md [hunk 1]
```diff
@@ -1 +1,54 @@
+# tables/variable_combinations
 
+the `variable_combinations` table lists out for a given variable combination what topics they belong to, what `geography_combinations` these are available for, and most importantly gives the `table_name` and row `name` that the raw data is avaiable at.
```
Comment (tomgiddingsjisc) [inline tables/variable_combinations.md]: Uppercase start to sentence

File: tables/variable_combinations.md [hunk 2]
```diff
@@ -1 +1,54 @@
+# tables/variable_combinations
 
+the `variable_combinations` table lists out for a given variable combination what topics they belong to, what `geography_combinations` these are available for, and most importantly gives the `table_name` and row `name` that the raw data is avaiable at. 
+## What are variable_combinations?
+
+`variable_combinations` is the table that brings together the data in the other tables that can then be used to reference the actual raw data in the other schema. the `table_name` and `name` fields are used to achieve this. However we must first identify which entries we are interested in. Below is an example
+
+## Example use
+
+in order to query this though one has to know the following information:
```
Comment (tomgiddingsjisc) [inline tables/variable_combinations.md]: Uppercase start

**Review by tomgiddingsjisc — COMMENTED** [review]
Only two typos I think!

File: tables/variable_combinations.md (hunk 1 above)
Comment (davidplumb) [reply tables/variable_combinations.md]: fixed

File: tables/variable_combinations.md (hunk 2 above)
Comment (davidplumb) [reply tables/variable_combinations.md]: fixed

**Review by tomgiddingsjisc — APPROVED** [review]
Looks good

**Pull request merged**